timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -8,6 +8,10 @@ types:
       base-type: string
       external:
         java: com.palantir.atlasdb.timelock.api.Namespace
+    GenericNamedMinTimestamp:
```

Comment:
Copied from https://github.com/palantir/atlasdb/pull/7341 to avoid PR chains/graphs and rebase/merge hell.